bigcache_test.go

```diff
@@ -106,7 +106,7 @@ func TestEntryNotFound(t *testing.T) {
 	_, err := cache.Get("nonExistingKey")
 
 	// then
-	assert.EqualError(t, err, "Entry \"nonExistingKey\" not found")
+	assert.EqualError(t, err, "Entry not found")
```

Comment:
`assert.EqualError(t, err, ErrEntryNotFound)` ?
and everything below 😉

Reply:
https://github.com/stretchr/testify/blob/master/assert/assertions.go#L1195
second argument should be string

Reply:
or what do you mean?

Reply:
Or change to `ErrEntryNotFound.Error()` ?

Reply:
Yes, `ErrEntryNotFound.Error()`. I'm sorry.